src/UpdatedPackagesCollector.js

```diff
@@ -151,6 +151,7 @@ export default class UpdatedPackagesCollector {
       return file.replace(folder + path.sep, "");
     });
 
+    // TODO: Ugh... something about this.
```

Comment:
instead of configuring groups, just configure each command independently. more work, but same outcome. unless i'm not understanding what you're trying to do.